docs/web/using-clients.mdx

```diff
@@ -110,21 +102,3 @@ Usage:
 ```ts
 await useClient(ElizaService).say({sentence: "I feel happy."});
 ```
-
-## Roll your own client
```

Comment:
Do you think it's worthwhile to link to our `custom-client` example here? https://github.com/connectrpc/examples-es/tree/main/custom-client? I know it's been a question in public Slack a couple times.

Reply:
I must have missed the cases in public slack, or I forgot. Maybe it's better to keep this section then.

The custom client example hasn't been updated yet. Taking a note to restore this section once it's updated.

Reply:
Restoring in https://github.com/connectrpc/connectrpc.com/pull/224.